jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/SeleniumTestHelper.java

```diff
@@ -131,4 +121,78 @@ public static void runSqlScript(final String sqlScript) throws IOException, Clas
             }
         }
     }
+
+    public static void checkServiceDeleteWasSuccessful(String serviceName, JschService jschSvc, Properties properties) {
+        LOGGER.info("checkServiceDeleteWasSuccessful {}", serviceName);
+
+        props = properties;
```

Comment:
I'm not really in favor of mixing static methods and class variables. If it can't be avoided we might just as well write a service class that does the JVM and web server deletion so that it adheres to the [single responsibility pattern](https://en.wikipedia.org/wiki/Single_responsibility_principle).